core-services/prow/02_config/_plugins.yaml

```diff
@@ -804,12 +804,36 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-4.9-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.6.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-misc-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.7.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-misc-configs
     ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.8.yaml:
       clusters:
         app.ci:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.9.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-misc-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.10.yaml:
```

Comment:
Is there a reason why all these patterns aren't a single `ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-*.yaml` key? That way you would not need our approval when adding new configs.

Same with job CMs below

Reply:
I believe we tried that before and got failures. I will try again.